AQ3DC/AQ3DC.py

```diff
@@ -309,34 +317,34 @@ def ComputationFolder(self):
         if computation_folder != "":
             self.ui.LineEditFolderComputation.setText(computation_folder)
 
-    def Computation(self):
+    def SaveComputationMeasuement(self):
```

Comment:
Making these names more descriptive is good, but it is confusing to see the capitalized names at the call site. For example in this code it looks like a class name is being used, but in reality it is a method:

```py
self.ui.ButtonCompute.clicked.connect(self.SaveComputationMeasurement)
```

Typically classes have capitalized names like `ScriptedLoadableModule` while functions would have lowercased names like `saveComputationMeasurement`.

I won't add a comment for every such function. I like the renaming in general but would prefer the standardized casing scheme so it is easier to tell at a glance which kinds of objects are being referenced.

Reply:
I understand, I will pull a new commit by renaming all the function.